monkey/monkey_island/cc/services/attack/technique_reports/T1210.py

```diff
@@ -27,20 +27,19 @@ def get_report_data():
 
 def get_scanned_services():
     results = mongo.db.telemetry.aggregate([{'$match': {'telem_type': 'scan'}},
-                                            {'$group': {
-                                                '_id': {'ip_addr': '$data.machine.ip_addr',
-                                                        'services': '$data.machine.services'
-                                                        },
-                                                'machine': {'$first': '$data.machine'}}}])
+                                           {'$sort': {'data.service_count': -1}},
```

Comment:
Is there any reason for this sort?

Reply:
Yes. One machine can get scanned by different monkeys. This attempts to get the scan that has the most services uncovered.